### PR TITLE
cloud: update connection page for TiDB Dedicated clusters

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/connect-to-tidb-cluster.md
Comment (Oreoxmt) [inline tidb-cloud/connect-to-tidb-cluster.md:1]: Preview: https://pingcap-docsite-preview-git-preview-cl-a2d682-oreoxmts-projects.vercel.app/tidbcloud/connect-to-tidb-cluster

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 1]
```diff
@@ -13,23 +13,28 @@ This document introduces the methods to connect to your TiDB Dedicated cluster.
 
 After your TiDB Dedicated cluster is created on TiDB Cloud, you can connect to it via one of the following methods:
 
-- [Connect via standard connection](/tidb-cloud/connect-via-standard-connection.md)
+- Direct connections
 
-    The standard connection exposes a public endpoint with traffic filters, so you can connect to your TiDB cluster via a SQL client from your laptop. You can connect to your TiDB clusters using TLS, which ensures the security of data transmission from your applications to TiDB clusters.
+    Direct connections use the MySQL native connection system over TCP. You can connect to your TiDB Dedicated cluster using any tool that supports MySQL connections, such as the [MySQL Command-Line Client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html). TiDB Cloud also provides [SQL Shell](/tidb-cloud/connect-via-sql-shell.md), which enables you to try TiDB SQL, test out TiDB's compatibility with MySQL quickly, and administer user privileges.
```
Comment (lilin90) [inline tidb-cloud/connect-to-tidb-cluster.md:18]: The meaning is correct, but it seems that "manage user privileges" is more commonly used.
```suggestion
    Direct connections use the MySQL native connection system over TCP. You can connect to your TiDB Dedicated cluster using any tool that supports MySQL connections, such as the [MySQL Command-Line Client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html). TiDB Cloud also provides [SQL Shell](/tidb-cloud/connect-via-sql-shell.md), which enables you to try TiDB SQL, test out TiDB's compatibility with MySQL quickly, and manage user privileges.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 2]
```diff
@@ -36,20 +36,18 @@ You can set the CIDR when creating the first TiDB Dedicated cluster. If you want
...
+    >     - 10.250.0.0 - 10.251.255.255
     >     - 172.16.0.0 - 172.31.255.255
     >     - 192.168.0.0 - 192.168.255.255
-
     > - For Google Cloud Region, it is recommended to configure an IP range size between `/19` and `/20`. If you want to configure an IP range size between `/16` and `/18`, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md). Supported network addresses include:
     >     - 10.250.0.0 - 10.251.255.255
     >     - 172.16.0.0 - 172.17.255.255
     >     - 172.30.0.0 - 172.31.255.255
-
     > - TiDB Cloud limits the number of TiDB Cloud nodes in a region of a project based on the CIDR block size of the region.
```
Comment (lilin90) [inline tidb-cloud/set-up-vpc-peering-connections.md:48]: FYI. My change to the preceding lines in the **Note** is to fix the display on the site.
image

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**